### ISSUE TITLE
🚀 add column placeholder in hyperlinks-file-link-format

### ISSUE THREAD

**Issue by eugenesvk** [dandavison/delta#1354]
Currently `hyperlinks-file-link-format` supports `{path}` and `{line}` placeholders.
However, the new file protocol (and many editors) also support columns
Would be great if `hyperlinks-file-link-format` added a `{column}` placeholder as well

**Comment by fabian-thomas** [comment]
Just want to add that `{host}` or `{hostname}` would be nice to have as well.

**Comment by dandavison** [comment]
> Just want to add that `{host}` or `{hostname}` would be nice to have as well.

@fabian-thomas I might be misunderstanding but I'm not sure a `{hostname}` placeholder makes sense? (What would it be populated with?) For example, my personal setting is currently

```gitconfig
   hyperlinks-file-link-format = "http://wormhole:7117/file/{path}:{line}?land-in=editor"
```

The hostname is something I hard-code into the link format template, and `{path}` and `{line}` are populated by delta as appropriate for the current file and line in delta output where the hyperlink occurs.

**Comment by fabian-thomas** [comment]
This typically only makes sense when you work on remote machines. I have setup my machine to open files on remotes via Emacs Tramp. The format is file://remote-hostname/path/to/file
Ripgrep has implemented the same, so one can probably steal some code there.

**Comment by dandavison** [comment]
@fabian-thomas So what you're proposing is that `{hostname}` would be populated with a system hostname, regardless of the diff input that delta is parsing, correct? This would be in contrast to `{path}` and `{line}` (and `{column}`) which are placeholders to be populated from the line of input being parsed by delta.

If that's correct, the alternative would be to hard-code that hostname in your git config `hyperlinks-file-link-format` -- could you discuss that alternative?

**Comment by fabian-thomas** [comment]
Yes, correct. I'm not entirely sure what you mean by the contrast to line and path. Let me explain my mental model of delta, maybe that helps. I think of delta as mainly two commands:
delta file1 file2
git diff (using delta)

For these two commands it makes sense to me to put the hostname in every hyperlink since all files reside on the machine and the hostname is clear. Like that I can easily jump to the corresponding diff.
Configuring the hostname via the git config would work, but is a pain when you deploy one set of dotfiles to each machine.
